packages/node/src/sdk.ts

```diff
@@ -149,7 +156,20 @@ export async function close(timeout?: number): Promise<boolean> {
 }
 
 /**
- * A function that returns a Sentry release string dynamically from env variables
+ * Function that checks if autoSessionTracking option is enabled
+ */
+export function isAutosessionTrackingEnabled(): boolean {
+  // Also add the checks that makes sure in case when you stop session tracking or resume
+  const client = getCurrentHub().getClient<NodeClient>();
+  const clientOptions: NodeOptions | null = client ? client.getOptions() : null;
```

Comment:
Adding `null` here looks unnecessarily complicated.

```suggestion
  const clientOptions = client && client.getOptions();
```